placement-rules.md

```diff
@@ -0,0 +1,142 @@
+
+## Specifying placement options
+
+Placement options can be specified using either _direct placement_ or using a _placement policy_. In the following example, both tables `t1` and `t2` have the same rules:
+
+```sql
+CREATE TABLE t1 (a INT) PRIMARY_REGION="us-east-1" REGIONS="us-east-1,us-west-1";
+CREATE PLACEMENT POLICY eastandwest PRIMARY_REGION="us-east-1" REGIONS="us-east-1,us-west-1";
+CREATE TABLE t2 (a INT) PLACEMENT POLICY=eastandwest;
+```
+
+Using placement policies is recommended to simplify rule management. Changes made to a placement policy (via [`ALTER PLACEMENT POLICY`](/sql-statements/sql-statement-alter-placement-policy.md)) automatically propagate to all database objects. This differs from direct placement options, which the rules must be altered for each object (tables, partitions).
+
+A `PLACEMENT POLICY` is not associated with any database schema and has global scope. Assigning a placement policy does not require any additional privileges over `CREATE TABLE` privilege.
+
+## Option reference
+
+> **Note:**
+>
+> Placement options depend on labels correctly specified in the configuration of each TiKV node. For example, `PRIMARY_REGION` depends on the label `region` in TiKV. To see a summary of all labels available in your TiKV cluster, use the statement [`SHOW PLACEMENT LABELS`](/sql-statements/sql-statement-show-placement-labels.md):
+>
+> ```sql
+> mysql> show placement labels;
+> +--------+----------------+
+> | Key    | Values         |
+> +--------+----------------+
+> | disk   | ["ssd"]        |
+> | region | ["us-east-1"]  |
+> | zone   | ["us-east-1a"] |
+> +--------+----------------+
+> 3 rows in set (0.00 sec)
+> ```
+
+| Option Name                | Description                                                                                    |
+|----------------------------|------------------------------------------------------------------------------------------------|
+| `PRIMARY_REGION`           | Raft leaders will be placed in stores which have the `region` label that match this value.     |
+| `REGIONS`                  | Raft followers will be placed in stores which have the `region` label that matches this value. |
+| `SCHEDULE`                 | The strategy used to schedule the placement of followers. Either `EVEN` (default) or `MAJORITY_IN_PRIMARY`. |
+| `FOLLOWERS`                | The number of followers. For example, `FOLLOWERS=2` means that there will be 3 copies of the data (2 followers and 1 leader). |
+| **Advanced configuration**                                                                                                  |
+| `CONSTRAINTS`              | A list of constraints that apply to all roles. For example, `CONSTRAINTS="[+disk=ssd]`.        |
+| `FOLLOWER_CONSTRAINTS`     | A list of constraints that only apply to followers.                                            |
+
```

Comment:
What about the clarification about feature incompatibilities and tools incompatibilities?

Reply:
Thank you for the suggestion. Can I get you to check out https://github.com/pingcap/docs/pull/6618/commits/dfdf1a3aa9fd43586eaa66c18a272f128b9f9dc7 :-) Thanks!